### ISSUE TITLE
Missing files from npm install?

### ISSUE THREAD

**Issue by davidalee** [ripple-emulator/ripple#27]
Is it possible to `npm install` this into an existing project? I tried that and noticed that not all of the necessary files were present.

**Comment by TimBarham** [comment]
Yes, it is definitely is possible to `npm install` into an existing project (Visual Studio used to do this, for example), but can you describe how the missing files are "necessary"? The npm package only includes files that are needed at runtime. The excluded files are only needed if you are developing / building *Ripple* itself.
